CarpetLib/src/interpolate_3d_3tl.cc

```diff
@@ -107,11 +107,10 @@ void interpolate_3d_3tl(T const *restrict const src1, CCTK_REAL const t1,
   RT const s2fac = (t - t1) * (t - t3) / ((t2 - t1) * (t2 - t3));
   RT const s3fac = (t - t1) * (t - t2) / ((t3 - t1) * (t3 - t2));
 
-// Loop over region
-#pragma omp parallel
+  // Loop over region
+  // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.